### PR TITLE
Fixed errors when there are undefined or null children.

### PR DESCRIPTION
**Pull request by limscoder**
Fixes exception when children are null or undefined.

### CODE REVIEW

**Comment by nelix** [comment]
Are numbers a problem too?

**Comment by limscoder** [comment]
I don't know, I didn't have that use case in the code I ran it against.

File: lib/assertions.js [hunk 1]
```diff
@@ -23,6 +23,10 @@ var hasChildTextNode = (props, children) => {
   React.Children.forEach(children, (child) => {
     if (hasText)
       return;
+    else if (child === null)
```
Comment (ryanflorence) [inline lib/assertions.js]: can we just do `(child == null)`?

**Comment by nelix** [comment]
Hmm, I ran these tests on an unmodified react-a11y and they pass?

**Comment by nelix** [comment]
I know this is a bug, but I was unable to make a test case that fails for some reason

**Comment by limscoder** [comment]
I'll be the old cliche: it worked (failed) on my machine

I was a little confused by the tests though. I couldn't get the test to fail without the `onClick={k}`, which was odd, but I'm used to using Jest. The use case seems simple enough, maybe someone who's more familiar with these tests can refactor into something better.

File: lib/assertions.js (hunk 1 above)
Comment (limscoder) [inline lib/assertions.js]: I like to be explicit, but that should work. Tests should be enough validation.

**Comment by nelix** [comment]
@ryanflorence could you provide some advice here?

**Comment by limscoder** [comment]
whichever way we need to fix this, the null/undefined issue is keeping us from using this cool project right now, and trying to maintain our own clone has a lot of overhead.

**Comment by AsaAyers** [comment]
@ryanflorence Is there something more that needs to be done before this can be fixed?

**Comment by AsaAyers** [comment]
Thanks :+1:

**Comment by ryanflorence** [comment]
`v0.0.6` is on npm now with this change. thanks :)

**Pull request merged**